addons/mail/static/src/components/chat_window/chat_window.js

```diff
@@ -240,6 +242,9 @@ class ChatWindow extends Component {
             this.chatWindow.focus();
         } else {
             this.chatWindow.fold();
+            if (this._threadRef.comp) {
```

Comment:
You had issue here without this condition? Feels like this should actually be the real condition to decide on the fold toggle (ie. using actual last rendered state and not model data, as I've suggested in the past).

In business terms, existence of `this._threadRef.comp` could be replaced with `_wasFolded` that is updating itself at mount/patch.

Reply:
Didn't check, just did the same code as in onShow/HideHomeMenu